website/docs/en/plugins/rspack/subresource-integrity-plugin.mdx

```diff
@@ -0,0 +1,184 @@
+import { ApiMeta } from '@components/ApiMeta.tsx';
+
+# SubresourceIntegrityPlugin
+
+<ApiMeta specific={['Rspack']} addedVersion="1.2.4" />
+
+The `rspack.experiments.SubresourceIntegrityPlugin` is a plugin for enabling Subresource Integrity in Rspack.
+
+> [Subresource Integrity](http://www.w3.org/TR/SRI/) (SRI) is a security
+> feature that enables browsers to verify that files they fetch (for
+> example, from a CDN) are delivered without unexpected
+> manipulation.
+
+## Features
+
+### Support for HTML Plugin
+
+The plugin supports integration with [`HtmlRspackPlugin`](/plugins/rspack/html-rspack-plugin.mdx) and [`html-webpack-plugin`](https://github.com/jantimon/html-webpack-plugin). It will automatically set the `integrity` and `crossorigin` attributes for the injected tags.
```

Comment:
```suggestion
The plugin supports integration with [`HtmlRspackPlugin`](/plugins/rspack/html-rspack-plugin) and [`html-webpack-plugin`](https://github.com/jantimon/html-webpack-plugin). It will automatically set the `integrity` and `crossorigin` attributes for the injected tags.
```